### PR TITLE
Create 2015-04-30-swagger-and-jrebel.md

### PR DESCRIPTION
**Pull request by TehShrike**
for @smithaps89 to continue

### CODE REVIEW

**Comment by saibotsivad** [comment]
:+1:

File: content/2015-04-30-swagger-and-jrebel.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+title: Getting JRebel to work while testing with Swagger
+date: Thu Apr 30 2015 13:23:22 GMT-0500 (CDT)
+
+Put your stuff here Andrew!
```
Comment (saibotsivad) [inline content/2015-04-30-swagger-and-jrebel.md]: There needs to be more content here. We can't publish it this way!

Comment (TehShrike) [inline content/2015-04-30-swagger-and-jrebel.md]: But Tobias, it's not actually published until the file name gets added to the [index.json](https://github.com/edatasource/tech-blog/blob/gh-pages/content/index.json) file anyway!

Comment (saibotsivad) [inline content/2015-04-30-swagger-and-jrebel.md]: Oh, good point!

**Comment by TehShrike** [comment]
I'm closing this because Andrew is creating a new file from his own branch.

**Pull request closed without merging**